docs/proposals/2170-kubeflow-training-v2/README.md

```diff
@@ -279,6 +279,43 @@ type TrainJob struct {
 	Status TrainJobStatus `json:"status,omitempty"`
 }
 
+const (
+	// TrainJobSuspended means the TrainJob is suspended.
+	TrainJobSuspended string = "Suspended"
+
+	// TrainJobCompleted means that the actual jobs have completed its execution.
```

Comment:
Should it be TrainJob, and similar for other statuses ?
```suggestion
	// TrainJobCompleted means that the TrainJob has completed its execution.
```

Reply:
Done.